Gruntfile.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['eslint','fh:test']);
```

Comment:
@wtrocki But if we need change the standard of the GruntJS file and tasks need be in another PR / JIRA. This JIRA is a task which is a part of one epic to add Eslint in the project and call it in the test task.